model/lib/email/presentation_email.dart

```diff
@@ -84,6 +89,10 @@ class PresentationEmail with EquatableMixin {
     sentAt,
     replyTo,
     preview,
-    hasAttachment
+    hasAttachment,
+    mailboxIds,
```

Comment:
why does props need to have these fields, IMO, i think that routeWeb contains mailboxId. And mailboxId is unique, so i think we don't need mailboxNames

Reply:
We need `mailboxNames` to display `label` for each email when searching